src/activity/events/events.css

```diff
@@ -0,0 +1,314 @@
+}
+
+.event-header {
+	margin: auto 0.5rem;
+	padding: 0.5rem;
+}
+
+/* date */
+.event-date {
+	display: inline-block;
+	margin: 0.5rem;
+	height: 80px;
+	width: 80px;
+	vertical-align: top;
+	text-align: center;
+	color: white;
+	font-weight: normal;
+	background-color: #21262d;
+	border: 2px solid rgb(33, 150, 243);
+	border-radius: 20px;
+	box-shadow: 0 0 10px rgba(33, 150, 243, 0.6);
+}
+
+.event-date p {
+	margin: auto;
+}
+
+.event-date #month {
+	padding: 5px;
+	font-size: 20px;
+}
+
+.event-date #day {
+	font-size: 36px;
+}
+
+/* Important info */
+.event-info {
+	margin: 0.5rem;
+	padding: 0 0.5rem;
+	max-width: 500px;
+	width: 90%;
+	display: inline-block;
+	min-height: 80px;
+	text-align: left;
+}
+
+.event-info #title {
+	margin: 0 0 10px 0;
+	padding: 0.5rem 0.75rem;
+	color: white;
+	font-size: 2.5vh;
+	font-size-adjust: inherit;
+	font-weight: normal;
+	color: white;
+	background-color: #21262d;
+	border-radius: 15px;
+	border: 2px solid rgb(33, 150, 243);
+	box-shadow: 0 0 10px rgba(33, 150, 243, 0.6);
+}
+
+.timeplace {
+	display: flex;
+	min-height: 36px;
+}
+
+.timeplace p {
+	margin: 5px;
+	padding: 4px 10px;
+	color: white;
+	background-color: #21262d;
+	border-radius: 10px;
+	vertical-align: middle;
+}
+
+/* Expand button */
+.expand-button {
+	margin: 0.5rem;
+	display: inline-flex;
+	height: 84px;
+	width: 84px;
+	justify-content: center;
+	vertical-align: top;
+}
+
+.expand-button button {
+	margin: auto;
+	width: 48px;
+	height: 48px;
+	border: 0;
+	border-radius: 50%;
+	cursor: pointer;
+	color: #ffa611;
+	background-color: #21262d;
+}
+
+#open {
+	transition: all 0.5s;
+	transform: rotate(-180deg);
+}
+
+#closed {
+	transition: all 0.5s;
+	transform: rotate(0deg);
+}
+
+.expand-button button:hover {
+	border: 3px solid #ffa611;
+}
+
+/* Expand content */
+.event-expand {
+	font-size: 18px;
+	color: white;
+	background-color: #21262d;
+	border-radius: 10px;
+	overflow: hidden;
+	transition: all 0.5s;
+}
+
+#expanded {
+	max-height: 800px;
+	margin: 0.5rem 1.5rem 2rem 1.5rem;
+	padding: 1.5rem;
+}
+
+#collapsed {
+	max-height: 0;
+	margin: 0 1.5rem;
+	padding: 0px 1.5rem;
+}
+
+.event-expand .details {
+	padding: 0 1rem;
+}
+
+.event-expand #meeting-link {
+	text-decoration: none;
+	color: #2d8cff;
+}
+
+.event-expand #poster-list {
+	display: inline-flex;
+	flex-direction: row;
+	margin: auto 10px;
+}
+
+.poster-img {
+	width: 150px;
+	height: 150px;
+	margin: auto 10px;
+	object-fit: cover;
+	border-radius: 10px;
+	box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.6);
+}
+
+.poster-img:hover {
+	opacity: 0.5;
+	transition: all 0.3s ease-in-out;
+}
+
+.poster-img:active {
+	background-color: white;
+}
+
+/* Responsive features*/
+@media only screen and (max-width: 800px) {
+	.events-header {
+		max-width: 360px;
+	}
+	.events-header h1 {
+		margin: 10px auto;
+		font-size: 24px;
+		border: 3px solid #ffa611;
+	}
+	.event {
+		display: flex;
+		flex-direction: column;
+	}
+
+	.event-date {
```

Comment:
Not sure if I like how far apart the month and day are when it's adapted to a smaller screen, but that's something we can talk about later.

Reply:
I'll figure the date layout for responsive mod before merging